commit 699db5a3a7b3ab30db711822d701de032bf14b51 (refs/changes/63/663/1)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2016-10-12T19:25:43-07:00 (3 years ago)
    
    CNS-167 Upgrade changefeed to 1.2.0 which contains the fix for AGENT-1041

diff --git a/package.json b/package.json
index 9992d5e..abef833 100644
--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
   "dependencies": {
     "assert-plus": "0.1.5",
     "bunyan": "1.5.1",
-    "changefeed": "1.1.9",
+    "changefeed": "1.2.0",
     "cueball": "1.0.2",
     "dashdash": "1.10.1",
     "deep-equal": "1.0.1",
